index.html

```diff
@@ -412,7 +412,7 @@
             The list of valid post properties is defined by the vocabularies. This allows new vocabularies to be developed outside the development of the syntax.
           </p>
           <p>
-            Most values will be strings. If a property (such as `author` for example) references another
+            Most values will be strings. If a property (such as <a>author</a> for example) references another
```

Comment:
Yeah, agreed. I'll send a PR with something that just looks reasonably good; it's easy enough to change later and we could bikeshed this all day